src/styles/landing-page.less

```diff
@@ -34,10 +34,10 @@
     top: @landing-offset-top;
     width: calc(100% ~"-" @landing-side-bar-width-sm);
   }
-  @media(min-width: @screen-md-min) {
+  @media(min-width: @screen-lg-min) {
     width: calc(100% ~"-" @landing-side-bar-width-md);
```

Comment:
Is this changing the sidebar width? We should name the variables so that `landing-side-bar-width-lg` is for `screen-lg-min`

cc @rhamilto since he originally set these values